tests/e2e/test_end_to_end.py

```diff
@@ -0,0 +1,2502 @@
+import os
+
+import intake
+import numpy as np
+import pytest
+
+from access_nri_intake.cli import build
+
+from .conftest import here
+
+e2e = pytest.mark.skipif(
+    "not config.getoption('--e2e')",
+)
+
+
+@e2e
+@pytest.fixture(scope="module")
+def current_catalog():
+    """
+    Return the current catalog as an intake catalog.
+    """
+    metacat = intake.cat.access_nri
+    yield metacat
+
+
+@e2e
+@pytest.fixture(scope="module")
+def metacat(BASE_DIR, v_num):
+    # Build our subset of the catalog. This should take ~2 minutes with the PBS
+    # flags in build_subset.sh
+    print(f"Building the catalog subset & writing to {BASE_DIR}")
+    print(f"Version number: {v_num}")
+    build(
+        [
+            f"{here}/configs/cmip5.yaml",
+            f"{here}/configs/access-om2.yaml",
```

Comment:
Two separate-but-related points:
- If `here` is being referenced directly, should it be a fixture like, e.g., `test_data` is?
- I think it's generally safer to build file paths using `os` rather than direct string building.